lib/ukboot/boot.c

```diff
@@ -339,6 +341,9 @@ void ukplat_entry(int argc, char *argv[])
 	uk_sched_start(s);
 #endif /* !CONFIG_LIBUKBOOT_NOSCHED */
 
+	ictx.cmdline.argc = argc - kern_args;
```

Comment:
Hmm, isn't `kern_args` always `0`? Ditto for line below.

Reply:
Ouh, this seems to come from a leftover before we updated `libuklibparam`. I can fix it with this commit. 👍🏻